idseq_pipeline/commands/host_filtering_functions.py

```diff
@@ -331,97 +339,99 @@ def fetch_genome_work(s3genome, strict):
                            "hg38_pantro5_k16"):
         write_to_log("Oh hello interesting new genome {}".format(genome_name))
     genome_dir = os.path.join(REF_DIR, genome_name)
+
     if not os.path.exists(genome_dir):
         try:
-            #TODO Clean up and fold into fetch_reference
-            # Hmm. I understand the desire to create a single function for fetching any reference we might need,
-            # but the functionality of these two functions is actually quite different.  One of them handles
-            # resiliently situations like the destination existing/not existing, being a folder/not a folder, etc.
-            # The other expands an archive into an existing folder, and needs to thoroughly cleanup in case
-            # something does not go as planned, because of the prefetching pattern of use.  Also, this function
-            # will fetch whichever of the "tar" or "tar.gz" versions of the archive actually exist in s3,
-            # even if the other version has been specified.  So the two functions are resilient in different ways.
             install_s3mi()
             tarfile = uncompressed(s3genome)
             try:
-                execute_command(
-                    "s3mi cat {tarfile} | tar xvf - -C {refdir}".format(
-                        tarfile=tarfile, refdir=REF_DIR))
+                print("Trying to download compressed genome...")
```

Comment:
should these be using logger / write_to_log?

Reply:
I believe they actually do the same thing throughout the program after the logger is configured, but write_to_log is probably better / changed it